### ISSUE TITLE
RIPE integration

### ISSUE THREAD

**Issue by plajjan** [SpriteLink/NIPAP#204]
Support for pushing updates to RIPE about inetnums, routes and so forth would be pretty sweet.

**Comment by fischa** [comment]
Why bother with a whois-wrapper. At least for RIPE and ARIN there are RESTful APIs for queries and updates of the DBs available:
https://github.com/RIPE-NCC/whois/wiki/WHOIS-REST-API
https://www.arin.net/resources/whoisrws/whois_api.html + http://projects.arin.net/arinr/arincli.7.html

**Comment by plajjan** [comment]
Good point :) although it only helps people in ARIN and RIPE land but I'm willing to live with that for now.

**Comment by fischa** [comment]
Well I'm aware that there are missing three other RIRs. But I'm pretty sure that you will have to write code for every RIR because there are always differences. Starting with RIPE NCC and ARIN would be a start. But there will always be differences between the systems... 

BTW: 
* LACNIC is working on a REST-whois service as well (http://www.lacnic.net/en/web/lacnic/documentos-tecnicos-whois)
* For APNIC I didn't find any hint on a potential REST-whois service
* AFRINIC has something in place as well (http://www.afrinic.net/en/library/news/1201-afrinic-launches-new-version-of-whois)
